knative-operator/build/Dockerfile

```diff
@@ -1,18 +0,0 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
-
-ENV OPERATOR=/usr/local/bin/knative-serving-openshift \
-    USER_UID=1001 \
-    USER_NAME=knative-serving-openshift
-
-# install operator binary
-COPY build/_output/bin/knative-serving-openshift ${OPERATOR}
-
-COPY build/bin /usr/local/bin
-RUN  /usr/local/bin/user_setup
-
-# install manifest[s]
-COPY deploy /deploy
-
-ENTRYPOINT ["/usr/local/bin/entrypoint"]
-
-USER ${USER_UID}
```

Comment:
I don't think this file is necessary anymore.

We're not using `operator-sdk build` for building the image.
This file actually creates some confusion and if we keep it to support `operator-sdk build` command, we need to make sure we sync it with upper level `Dockerfile`.